test/drivers/test_suites/generic_tracepoints_suite/sched_process_exec.cpp

```diff
@@ -217,9 +220,21 @@ TEST(GenericTracepoints, sched_proc_exec_success_memfd)
 	 */
 	evt_test->assert_numeric_param(20, (uint32_t)PPM_EXE_WRITABLE | PPM_EXE_FROM_MEMFD);
 
+	/* Parameter 28: trusted_exepath (type: PT_FSPATH) */
+	/* In the kmod we use d_path helper so case like memfd are correcly managed */
+	if(evt_test->is_kmod_engine())
+	{
+		evt_test->assert_charbuf_param(28, "/memfd:malware (deleted)");
+	}
+	else
+	{
+		/* In BPF drivers we have no the correct result but we can reconstruct part of it */
+		evt_test->assert_charbuf_param(28, "memfd:malware");
+	}
```

Comment:
Unfortunately, the final outputs are different but we cannot handle this complexity in the bpf drivers, btw the result seems acceptable